Dockerfile

```diff
@@ -118,7 +118,7 @@ RUN ${APPLIANCE_ROOT}/setup && \
 WORKDIR ${APP_ROOT}
 RUN source /etc/default/evm && \
     export RAILS_USE_MEMORY_STORE="true" && \
-    npm install gulp bower -g && \
+    npm install gulp bower yarn -g && \
```

Comment:
This section isn't just for SUI.  We do 'npm install' just once in Dockerfile.  This is same as how it's done in kickstart file (https://github.com/ManageIQ/manageiq-appliance-build/blob/78159024360dd9c0c3a978326879de3c00892866/kickstarts/partials/post/ui_compile.ks.erb#L1)